features/minion/repository/pom.xml

```diff
@@ -74,7 +74,13 @@
                                 <feature>minion-health-check</feature>
                                 <feature>minion-api-layer</feature>
                                 <feature>minion-plugin-sample</feature>
-                                <feature>opennms-deviceconfig</feature>
+                                <feature>opennms-deviceconfig-deps</feature>
```

Comment:
Shrink this to one module for sink and one module for monitor

Reply:
There is now one feature for sink and one for monitor. But there are still some more (tftp and deps). Deps should be scrutinized to reduce it to a minimum.

Reply:
Removed the transitive features.